meilisearch-release-changelog/v0.30.0.md

```diff
@@ -0,0 +1,257 @@
+# v0.30.0 release changelogs
+
+<!-- The following line should NOT be put in the official release changelogs -->
+⚠️ Since this is a release candidate (RC), we do NOT recommend using it in a production environment. Something not working as expected? We welcome [bug reports](https://github.com/meilisearch/meilisearch/issues/new/choose) and [feedback about new features](https://github.com/meilisearch/product/discussions).
+
+This release introduces index deployment with zero downtime and adds task management features. It also brings the capability to navigate search results by page selection.
+
+Thanks to Hacktoberfest, tons of updates and bug fixes have been done by the fantastic Meilisearch community. You were more present than ever and words cannot do justice tojust how grateful we are for your support. A big thank you from the whole Meilisearch team! ❤️
+
+<!-- The following line should NOT be put in the official release changelogs -->
+📖 Meilisearch's official documentation will be ready for the official v0.30.0 release. Meanwhile, work-in-progress documentation is available [here]().
+
+# New features and improvements 🔥
+
+## Improve search result navigation: exhaustive number of search results
+
+When paginating search results with `offset` and `limit`, Meilisearch only returns an estimate of the total number of results. Since `estimatedTotalHits` can change, creating reliable pagination interfaces with numbered page selectors is challenging.
+
+v0.30 introduces two new search parameters, `page` and `hitsPerPage`. Queries with these parameters return an exhaustive number of `totalHits` and `totalPages`, which you can then use to create UI elements such as numbered page selectors.
+
+The following example fetches the second page of results for a given query:
+
+```bash
+curl \
+  -X POST 'http://localhost:7700/indexes/movies/search' \
+  -H 'Content-Type: application/json' \
+  --data-binary '{ "q": "shifu", "page": "2", "hitsPerPage": "10" }'
+```
+
+```json
+{
+    "hits": [
+        // … 10 hits
+    ],
+    // …
+    "page": 2,
+    "hitsPerPage": 10,
+    "totalHits": 2100,
+    "totalPages": 210
+}
+```
+
+Done in #2601 by @ManyTheFish.
+
+## Cancel processing and enqueued tasks
+
+Meilisearch now allows you to cancel `enqueued` or `processing` tasks through a new API route, `/tasks/cancel`. This can be useful when you need to interrupt a task that is taking too much time to be completed or enqueued by mistake.
+
+To cancel tasks, filter them by specifying one of the following query parameters: `uids`, `statuses`, `types`, or `indexUids`.
+
+Tasks can be filtered by date fields:
+- `beforeEnqueuedAt` / `afterEnqueuedAt`
+- `beforeStartedAt` / `afterStartedAt`
+- `beforeFinishedAt` / `afterFinishedAt`
```

Comment:
```suggestion
```

You could decide to omit it as the doc team [did](https://github.com/meilisearch/documentation/blob/7ddd504cdc7dd5834931c513b6ac3fb90bfad2eb/reference/api/tasks.md#query-parameters-1). 🤓